packages/ui/lib/RadioGroup/index.tsx

```diff
@@ -0,0 +1,50 @@
+import { forwardRef, HTMLAttributes } from "react";
+
+import { RadioGroupContext } from "./context";
+import {
+  RadioGroupItemRefProps,
+  RadioGroupProps,
+  useRadioGroup,
+} from "./useRadioGroup";
+import { RadioGroupItemProps, useRadioGroupItem } from "./useRadioGroupItem";
+
+const Group = forwardRef<
+  HTMLDivElement,
+  RadioGroupProps<HTMLAttributes<HTMLDivElement>>
+>((props, forwardedRef) => {
+  const { ctx, radioGroupProps, radioGroupRef } = useRadioGroup({
+    props,
+    forwardedRef,
+  });
+  return (
+    <RadioGroupContext.Provider value={ctx}>
```

Comment:
For this I wonder if we could also just add this provider to radioGroupProps.children and just have:

```tsx
return (
  <div {...radioGroupProps} ref={radioGroupRef} />
)

Not 100% confident atm, but it might be an option.